resources/webviews/aksclusterproperties/clusterproperties.html

```diff
@@ -7,6 +7,34 @@
     <title>AKS Cluster Properties</title>
     <!-- Link to the css file -->
     <link rel="stylesheet" href="{{cssuri}}">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://use.fontawesome.com/7dddd54b5c.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
+        integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+
+    <script>
+        // https://code.visualstudio.com/api/extension-guides/webview#passing-messages-from-a-webview-to-an-extension
+        const vscode = acquireVsCodeApi();
```

Comment:
The docs in the link say "For security reasons, you must keep the VS Code API object private and make sure it is never leaked into the global scope."

Could we move it inside the document-ready function?